app/src/main/java/com/randomappsinc/studentpicker/home/MainActivity.java

```diff
@@ -174,6 +177,7 @@ public void addItem() {
             nameListsAdapter.addList(newList);
             Intent intent = new Intent(this, ListActivity.class);
             intent.putExtra(LIST_NAME_KEY, newList);
+            intent.putExtra(LIST_TYPE, START_ON_EDIT_PAGE);
```

Comment:
Sorry for the miscommunication; I meant that the key should be:

`public static final String START_ON_EDIT_PAGE = "startOnEditPage"`

And we just pass in true as the value here.